internal/initwd/test-fixtures/invalid-version-constraint/root.tf

```diff
@@ -0,0 +1,7 @@
+# This fixture depends on a github repo at:
+#     https://github.com/hashicorp/terraform-aws-module-installer-acctest
```

Comment:
If this were true then we'd need to put the `TF_ACC=1` guard at the start of the test to ensure it isn't included in a normal test run (as you can see in e.g. `TestLoaderInstallModules_registry`.

However, I think this isn't actually true in practice, right? Because the presence of the version constraint will block Terraform from ever attempting to fetch this repository. If that's so, I think it'd be helpful to future maintainers for the comment to explain that the address here is never actually fetched because of the error.